### ISSUE TITLE
Problem with loss

### ISSUE THREAD

**Issue by HeVLF** [cnuzh/LG-GNN#3]
Hello, thank you for sharing your work! I am trying to run your code but I'm encountering some errors with computing the loss, mainly due to the majority of the epochs having a loss equal to "inf" or even "nan". So I am not sure if I changed something without noticing. But would love to hear from you or someone that has faced the same issue.

Best.

**Comment by cnuzh** [comment]
@HeVLF Hi, have you normalized the data you entered? Is it cross-entropy loss or mutual information loss?  Is the weight of the loss function set correctly?

**Comment by HeVLF** [comment]
The situation that I described was with ABIDE and the default setting by you, so with both cross entropy and mutual information, with the weight being equal to 0.1. After I tried with only cross entropy and the results were what one would expect. But I still couldn’t find the error using the loss as described in your paper.
Thank you for your prompt response.

**Comment by cnuzh** [comment]
The output should be normalized

**Comment by sws827767542** [comment]
您好，请问我在运行的时候为什么train spe: 0.00000
